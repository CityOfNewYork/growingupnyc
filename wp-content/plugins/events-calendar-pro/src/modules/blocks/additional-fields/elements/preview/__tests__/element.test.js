/**
 * External dependencies
 */
import React from 'react';

/**
 * Internal dependencies
 */
import Preview from '../element';

describe( '<Preview>', () => {
	test( 'default params', () => {
		const component = renderer.create(
			<Preview name={ 'Modern Tribe' }>
				The Next Generation of Digital Agency
			</Preview>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
