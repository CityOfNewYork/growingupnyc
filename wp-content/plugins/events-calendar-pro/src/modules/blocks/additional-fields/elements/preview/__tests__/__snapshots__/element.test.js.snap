// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preview> default params 1`] = `
<div
  className="tribe-editor__additional-fields__preview"
>
  <h2
    className="tribe-editor__heading tribe-editor__heading--h2 tribe-editor__additional-fields__preview-title"
  >
    Modern Tribe
  </h2>
  <p
    className="tribe-editor__paragraph tribe-editor__paragraph--medium"
  >
    The Next Generation of Digital Agency
  </p>
</div>
`;
